test/sharness/t0027-rotate.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+test_description="Test rotate command"
+
+. lib/test-lib.sh
+
+test_init_ipfs
+
+test_expect_success "Save first ID and key" '
+ipfs id -f="<id>" > first_id &&
+ipfs id -f="<pubkey>" > first_key
+'
+
+test_launch_ipfs_daemon
+
+test_kill_ipfs_daemon
+
+test_expect_success "rotating keys" '
+ipfs rotate --oldkey=oldkey
+'
```

Comment:
Maybe we should put this in a function like you did in the init PR, so we can verify RSA, Ed25519 and the default all work.